webdriver/tests/bidi/network/get_data/request.py

```diff
@@ -20,10 +20,41 @@ async def test_request_text_file(
 
 
 async def test_request_base64_file(
-    bidi_session, url, setup_collected_response,
+    bidi_session,
+    url,
+    setup_collected_response,
 ):
     [request, _] = await setup_collected_response(fetch_url=url(PAGE_EMPTY_IMAGE))
     data = await bidi_session.network.get_data(request=request, data_type="response")
 
     assert data["type"] == "base64"
     assert isinstance(data["value"], str)
+
+
+@pytest.mark.parametrize(
+    "charset", [None, "utf-8", "iso-8859-15"]
```

Comment:
I assume for incorrectly set charsets we fallback to binary as well. Should we add some invalid cases as well?

Reply:
Sure we can add a test with an invalid charset. Note we don't fallback to binary for invalid charsets, we should just always use utf-8 to serialize to protocol bytes.